### PR TITLE
review-controller-and-view

### PR DESCRIPTION
**Pull request by richardhaenel**

### CODE REVIEW

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class ReviewsController < ApplicationController
+  def create
+    @studio = studio.find(params[:studio_id])
+    @review = Review.new(review_params)
+    @review.studio = @studio
+    if @review.save
+      redirect_to studio_path(@studio)
+    else
+      @booking = Booking.new
```
Comment (kmontes417) [inline app/controllers/reviews_controller.rb]: not sure

File: app/controllers/reviews_controller.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+class ReviewsController < ApplicationController
+  def create
+    @studio = studio.find(params[:studio_id])
```
Comment (kmontes417) [inline app/controllers/reviews_controller.rb]: @booking

File: app/controllers/reviews_controller.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    @review = Review.new(review_params)
+    @review.studio = @studio
```
Comment (kmontes417) [inline app/controllers/reviews_controller.rb]: @booking

File: app/controllers/reviews_controller.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    if @review.save
+      redirect_to studio_path(@studio)
```
Comment (kmontes417) [inline app/controllers/reviews_controller.rb]: @booking.studio

File: app/views/reviews/_new.html.erb [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<h3>Add a review</h3>
+<%= simple_form_for [studio, review] do |f| %>
```
Comment (kmontes417) [inline app/views/reviews/_new.html.erb]: [@booking, @review]

File: app/controllers/reviews_controller.rb (hunk 2 above)
Comment (richardhaenel) [reply app/controllers/reviews_controller.rb]: thanks, luigi

**Pull request merged**